recipe_scrapers/vegolosi.py

```diff
@@ -7,24 +7,3 @@ class Vegolosi(AbstractScraper):
     @classmethod
     def host(cls):
         return "vegolosi.it"
```

Comment:
I think we should restore this (even though it does theoretically work from the `schema` data.. let's leave as-is in this changeset).

```suggestion
        return "vegolosi.it"

    def title(self):
        return self.soup.h1.get_text().strip()
```